tests/tests.rs

```diff
@@ -98,10 +98,17 @@ fn create_bindgen_builder(header: &PathBuf)
     let header_str = try!(header.to_str()
         .ok_or(Error::new(ErrorKind::Other, "Invalid header file name")));
 
+    let objc_guard;
+    if header_str.contains("objc_") {
```

Comment:
instead of this, let's add a new line to the test configuration (see the `if line.contains(bindgen-flags` check above). `// bindgen-osx-only` maybe?